generic_relations/relations.py

```diff
@@ -42,6 +42,9 @@ def field_to_native(self, obj, field_name):
         Delegates to the `to_native` method of the serializer registered
         under obj.__class__
         """
+        if not (obj and getattr(obj, field_name)):
```

Comment:
This is perhaps a second case, but i just came across a situation where `obj` is passed into this method as None as well.  i haven't even figured out how that happened yet, or i'd be making a test for it...  in either case, this if statement should handle both cases.